apps/finance/app/package.json

```diff
@@ -7,10 +7,11 @@
     "@aragon/api": "^1.0.0",
     "@aragon/api-react": "^1.0.0-beta.2",
     "@aragon/templates-tokens": "^1.2.0",
-    "@aragon/ui": "^0.34.0",
+    "@aragon/ui": "^0.37.0",
     "@babel/polyfill": "^7.0.0",
     "bn.js": "^4.11.8",
     "date-fns": "2.0.0-alpha.22",
+    "downloadjs": "^1.4.7",
```

Comment:
Looking at this dependency, is it mainly to smooth over some iOS incompatibilities (I think we were having some issues before)?

Reply:
I added this dep to be able to download the file dynamically. Before the dep it was being downloaded via the `download` + `href` props of a link - but this is rendered with the render context, if the user changes any custom label that does not trigger a re-render and a download will not have all the custom labels updated. With this dep, the file contents get created on demand to make sure all the custom labels exported are up to date.